src/modules/roc_netio/target_libuv/roc_netio/udp_sender_port.cpp

```diff
@@ -98,6 +100,9 @@ bool UdpSenderPort::open() {
         return false;
     }
 
+    const int fd_err = uv_fileno((uv_handle_t*)&handle_, &fd_);
+    roc_panic_if(fd_err != 0);
```

Comment:
Would be nice to panic with meaningful error message with the error and strerror, like:

```
roc_panic("udp sender: uv_fileno(): [%s] %s", ...)
```